app/service/analytics.py

```diff
@@ -21,17 +21,17 @@
 
 
 @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
-def summary() -> SummaryDTO:
+def summary(user_email: str) -> SummaryDTO:
     """
     Gets an analytics summary from the repository.
 
     :return SummaryDTO: The analytics summary found.
     """
     try:
-        n_collections = collection_service.count()
-        n_families = family_service.count()
-        n_documents = document_service.count()
-        n_events = event_service.count()
+        n_collections = collection_service.count(user_email)
```

Comment:
Hmm, why isn't `restrict_entities_to_user_org` called from here only once... rather than 'n' times in the functions that follow. Also I'd be happy if this service went directly to the repo levels for the dfce entities - thoughts?

Reply:
Resolved